### PR TITLE
Helm: RBAC Chart

### PR DESCRIPTION
**Pull request by djsd123**
Initial commit

A chart to deploy miscellaneous `rbac` resources for users (mainly)

[Trello](https://trello.com/c/ZK7eHGqB)

### CODE REVIEW

File: charts/rbac/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# RBAC Resources
+
+Chart to deploy miscellaneous RBAC resources
+
+### Installing and Upgrading a release
+
+```
+helm upgrade rbac-chart analytics-platform-helm-charts/charts/rbac --install
+```
```
Comment (andyhd) [inline charts/rbac/README.md]: This readme is a bit terse. What are the miscellaneous RBAC resources? Why might I want to install this chart?

File: charts/rbac/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{{/* vim: set filetype=mustache: */}}
```
Comment (andyhd) [inline charts/rbac/templates/_helpers.tpl]: Even though I am a die-hard vim fan, I think you should keep your settings in your `.vimrc` file and not have them here.

Comment (andyhd) [reply charts/rbac/templates/_helpers.tpl]: Not to mention that this is not a mustache file.

File: charts/rbac/templates/_helpers.tpl [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Expand the name of the chart.
+*/}}
+{{- define "rbac.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "rbac.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
```
Comment (andyhd) [inline charts/rbac/templates/_helpers.tpl]: Is this algorithm a standard? If the chart name is too long, the version will be truncated first. That seems wrong.

**Comment by djsd123** [comment]
@andyhd @davidread @r4vi  ^^

**Pull request merged**